kroxylicious-api/src/main/java/io/kroxylicious/proxy/config/tls/TlsClientAuth.java

```diff
@@ -0,0 +1,25 @@
+package io.kroxylicious.proxy.config.tls;
+
+public enum TlsClientAuth {
+
+    REQUIRED("required", "REQUIRE"),
+    REQUESTED("requested", "OPTIONAL"),
+    NONE("none", "NONE");
+
+    private final String clientAuth;
+    private final String nettyClientAuth;
+
+    TlsClientAuth(String clientAuth, String nettyClientAuth) {
+        this.clientAuth = clientAuth;
+        this.nettyClientAuth = nettyClientAuth;
+    }
+
+    public String getClientAuth() {
```

Comment:
I think these methods and fields (clientAuth and nettyClientAuth) of the enum are now dead.